### PR TITLE
RDAPI-16735: Cassandra Admin sanity check

### PR DESCRIPTION
**Pull request by andreamussap**

### CODE REVIEW

File: content/en/docs/cass_admin/_index.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
 title: "API Gateway Apache Cassandra Administrator Guide"
 linkTitle: "Administer Apache Cassandra"
 weight: 9
-date: 2019-06-05T00:00:00.000Z
+date: 2019-06-05
```
Comment (alexearnshaw) [inline content/en/docs/cass_admin/_index.md]: Just FYI for future ref it's netlify CMS that changes the date format so there's no need to change it back as netlify will only change it again

File: content/en/docs/cass_admin/cassandra_architecture.md [hunk 2]
```diff
@@ -4,157 +4,113 @@ linkTitle: "Cassandra deployment architectures"
...
-storage.
+Multiple API Gateway groups can also be deployed on the same host, each host running an API Gateway instance for each group. This applies to both local and remote storage.
 
 {{< alert title="Note" >}}
 Windows is supported only for a limited set of developer tools. API Gateway and API Manager do not support Windows. You can continue using an existing remote Cassandra installation running
 on Windows that you have configured with your API Gateway installation.
 {{< /alert>}}
 
-For details on hosting Cassandra in datacenters, see [Configure API Management in multiple datacenters](/csh?context=303&product=prod-api-gateway-77) in the [API Gateway Installation Guide](/bundle/APIGateway_77_InstallationGuide_allOS_en_HTML5/).
+For details on hosting Cassandra in datacenters, see [Configure API Management in multiple datacenters](/bundle/APIGateway_77_InstallationGuide_allOS_en_HTML5/page/Content/InstallGuideTopics/TemplateTopics/post-install/post_multi-datacenter.htm).
```
Comment (alexearnshaw) [inline content/en/docs/cass_admin/cassandra_architecture.md]: Also there's no value in changing these links (csh links to other guides) right now. We will need to change them all together when everything is migrated.

File: content/en/docs/cass_admin/cassandra_bur.md [hunk 3]
```diff
@@ -68,58 +47,62 @@ You can do this using `cqlsh` or `kpsadmin` commands:
 
 Using `cqlsh`, execute the following command:
 
-```cql
+```
```
Comment (alexearnshaw) [inline content/en/docs/cass_admin/cassandra_bur.md]: Wondering why you removed 'cql' after the backticks? It tells the markdown parser to syntax colour this example for CQL programming language.

Comment (andreamussap) [reply content/en/docs/cass_admin/cassandra_bur.md]: I removed just to be consistent with 99% of our code blocks, which don't show the language!??

Comment (alexearnshaw) [reply content/en/docs/cass_admin/cassandra_bur.md]: Ah ok, I think it's good to use it when we can as the syntax highlighting is helpful especially for longer samples. Sometimes it's hard to know which language code to apply though that's why I haven't added it in most code blocks.
No need to add this one back as it's not a long sample anyway

**Pull request merged**